torch/_inductor/ir.py

```diff
@@ -2271,7 +2271,9 @@ def _normalize_size(x, new_size):
             if new_size[i] == -1:
                 assert old_size[i] is not None
                 new_size[i] = old_size[i]
-            elif old_size[i] is None or old_size[i] == 1:
+            elif old_size[i] is None or V.graph.sizevars.shape_env.evaluate_expr(
```

Comment:
welp, I can't actually use evaluate_expr() if the expressions has unbacked symints in it

Reply:
ah ok, I think I just want to guard with `size_oblivious=True` (I think this seems... reasonable for cases where inductor is dealing with broadcasting?)

Reply:
Yes, size oblivious is the normal fix here, but you /really/ want to be operating on SymInts here now lol